src/main/java/com/dream11/rest/provider/JsonProvider.java

```diff
@@ -27,6 +28,10 @@ public JsonProvider() {
     this.setMapper(DatabindCodec.mapper());
   }
 
+  public JsonProvider(ObjectMapper mapper) {
```

Comment:
Let's get rid of no argument constructor

Reply:
done